docs/tutorials/my-first-custom-action.rst

```diff
@@ -0,0 +1,255 @@
+
+    def _report_failed_scheduling(event: EventEvent, pod_name: str, message: str):
+        # this is how you send data to slack or other destinations
+        event.add_enrichment([
+            MarkdownBlock(f"Failed to schedule a pod named '{pod_name}', error: {message}"),
+        ])
+
+Before we proceed, we need to enable local playbook repositories in Robusta.
+
+Follow this quick guide to learn how to package your python file for Robusta: :ref:`Custom playbook repositories`
+
+Use this useful debugging commands to make sure your action ( ``report_scheduling_failure``) is loaded:
+
+.. code-block:: bash
+
+    robusta logs # get robusta logs, see errors.
+    robusta playbooks list-dirs  # get see if you custom action was loaded
+
+Let’s push the new action to Robusta, and then test it by triggering the action manually immediately.
+
+.. code-block:: bash
+
+    robusta playbooks push <PATH_TO_LOCAL_PLAYBOOK_FOLDER>
+    robusta playbooks trigger report_scheduling_failure name=robusta-runner-8cd69f7cb-g5bkb namespace=default seconds=5
+
+Check our slack channel, and:
+
+.. image:: /images/example_report_scheduling_failure.png
+
+Connection the trigger to the action - a Playbook is born!
+-------------------------------------
+
+We need to add a custom playbook that this action it in the generated_values.yaml.
+
+.. code-block:: yaml
+
+    globalConfig:
+      signing_key: XXXX
+      account_id: XXXX
+    sinksConfig:
+    - slack_sink:
+        name: main_slack_sink
+        slack_channel: '#my-slack-channel'
+        api_key: XXXXX
+    - robusta_sink:
+        name: robusta_ui_sink
+        token: XXXXXX  # generated with `robusta gen-config`
+    clusterName: kind-kind
+    enablePrometheusStack: true
+    kubewatch:
+      resources:
+        requests:
+          memory: 64Mi
+    grafanaRenderer:
+      resources:
+        requests:
+          memory: 64Mi
+    runner:
+      resources:
+        requests:
+          memory: 778Mi
+    # Custom Playbooks from here
+    customPlaybooks:
+    - triggers:
+      - on_event_create: {}
+      actions:
+      - report_scheduling_failure: {}
+    # Enable loading playbooks to a persistent volume
+    playbooksPersistentVolume: true
+
+.. note::
+    If you haven't already, make sure to clean up the pod from the last section by running ``kubectl delete pod nginx``
+
+Time to update Robusta’s config with the new generated_config.yaml:
+
+.. code-block:: bash
+
+    helm upgrade robusta robusta/robusta --values=generated_values.yaml
+
+After a minute or two Robusta will be ready.
+
+Let’s push the new action to Robusta:
+
+.. code-block:: bash
+    robusta playbooks push <PATH_TO_PLAYBOOK_FOLDER>
+
+After a minute or two Robusta will be ready.
+
+Great!
+
+Run the scenario from the first section again (creating a bad bad configuration), and you should see this in your slack:
+
+Check our slack channel, and:
+
+.. image:: /images/example_report_scheduling_failure.png
+
+
```

Comment:
Should we add cleanup commands. 
Like, how to clean everything we installed for the tutorial